src/Symfony/Component/Config/Tests/Builder/Fixtures/AddToList/Symfony/Config/AddToList/Messenger/RoutingConfig.php

```diff
@@ -15,10 +15,11 @@ class RoutingConfig
     private $senders;
     
     /**
-     * @param ParamConfigurator|list<mixed|ParamConfigurator> $value
+     * @param ParamConfigurator|list<ParamConfigurator|mixed> $value
+     *
```

Comment:
I'm seeing that this blank line is missing on other methods, it could be nice to have it everywhere.
Also, the order of the items in the union makes more sense to me in 6.0, it'd be nice to have the same in 5.4 I suppose.